independent-publisher-connectors/ScrapingBee/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1407 @@
+                  "title": "Used API Credit"
+                },
+                "max_concurrency": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "The max concurrency.",
+                  "title": "Max Concurrency"
+                },
+                "current_concurrency": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "The current concurrency.",
+                  "title": "Current Concurrency"
+                },
+                "renewal_subscription_date": {
+                  "type": "string",
+                  "description": "The renewal subscription date.",
+                  "title": "Renewal Subscription Date"
+                }
+              }
+            }
+          }
+        },
+        "description": "Retrieve information about credit consumption and concurrency usage."
+      }
+    },
+    "/v1/store/google": {
+      "get": {
+        "summary": "Perform Google search",
+        "tags": [
+          "Google Search API"
+        ],
+        "operationId": "SimpleSearch",
+        "deprecated": false,
+        "parameters": [
+          {
+            "name": "search",
+            "in": "query",
+            "x-ms-summary": "Search",
+            "required": true,
+            "type": "string",
+            "description": "The text you would put in the Google search bar."
+          },
+          {
+            "name": "country_code",
+            "in": "query",
+            "x-ms-summary": "Country Code",
+            "required": false,
+            "type": "string",
+            "description": "The country you would like the request to come from."
+          },
+          {
+            "name": "nb_results",
+            "in": "query",
+            "x-ms-summary": "Results",
+            "required": false,
+            "type": "integer",
+            "format": "int32",
+            "description": "The number of results to return."
+          },
+          {
+            "name": "page",
+            "in": "query",
+            "x-ms-summary": "Page",
+            "required": false,
+            "type": "integer",
+            "format": "int32",
+            "description": "The page number to extract results from."
+          },
+          {
+            "name": "language",
+            "in": "query",
+            "x-ms-summary": "Language",
+            "required": false,
+            "type": "string",
+            "description": "The language to return the results in."
+          },
+          {
+            "name": "extra_params",
+            "in": "query",
+            "x-ms-summary": "Extra Params",
+            "required": false,
+            "type": "string",
+            "description": "Any additional URL parameters to submit."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The .",

```

Comment:
@troystaylor can you please double check the description here! it looks like it is incomplete.